boltstub/wiring.py

```diff
@@ -129,6 +135,99 @@ def __str__(self):
         return "[{}]:{}".format(*self)
 
 
+class RegularSocket:
+    """A socket with an already receive cached value."""
+
+    def __init__(self, socket_, cache_) -> None:
+        self._socket = socket_
+        self._cache = cache_
```

Comment:
```suggestion
    def __init__(self, socket_, cache) -> None:
        self._socket = socket_
        self._cache = cache
```
`cache` is not shadowing anything.